molecule/alternative/playbook.yml

```diff
@@ -12,3 +10,5 @@
       - "/srv/data2"
       - "/srv/data3"
       - "/srv/data4"
+  roles:
```

Comment:
Why swap order of `roles` and `vars`? This is a dict/map so from programming point of view it doesn't change anything so we can only take an ease of code reading in account. Considering that `roles` key won't increase in size, but `vars` can, it would be better to have `roles` before `vars` as this won't cause problems in reading complex `vars` entries and simplify addition of new entries to this file.

Reply:
I swapped that back.
The reason is that usually the vars refers to the declared roles. I think visually it's better to have the vars declared before the roles.
But I swapped them back in 2de48ef

Reply:
Yeah, it is usually a matter of personal preference and usually with such things I look what guys from OpenStack-Ansible use since they have more written guidelines than ansible itself. In this case they use `vars` after `roles`, like here: https://github.com/openstack/openstack-ansible/blob/master/playbooks/infra-journal-remote.yml

Reply:
Indeed :)